packages/server/src/entity/QuestionModel.ts

```diff
@@ -17,29 +19,29 @@ export class QuestionModel extends BaseEntity {
 
   @ManyToOne((type) => QueueModel, (q) => q.questions)
   @JoinColumn({ name: "queueId" })
-  queue: Promise<QueueModel>;
+  queue: QueueModel;
 
   @Column({ nullable: true })
   queueId: number;
 
   @Column("text")
   text: string;
 
-  @ManyToOne((type) => UserCourseModel)
+  @ManyToOne((type) => UserModel)
   @JoinColumn({ name: "creatorId" })
-  creator: Promise<UserCourseModel>;
+  creator: UserModel;
```

Comment:
Ah, disregard previous comment, as I see that now the types should line up with this change.

Was this a mistake we had made earlier?

Reply:
yeah it wasn't exactly a mistake but not optimal to be using the UserCourseModel